examples/simple_cnn.py

```diff
@@ -46,20 +46,23 @@
     tf_model.evaluate(x_test, y_test)
     print("TF evaluation:%f" % (perf_counter() - tf_eval_start_time))
 
-    # Create, suitable converter to convert TF model to ML GeNN
-    converter = (FewSpike(K=8, norm_data=[x_norm]) if args.few_spike 
-                 else RateBased(input_type=args.input_type, 
-                                norm_data=[x_norm],
-                                norm_method=args.norm_method,
-                                spike_norm_time=500))
+    # Create a suitable converter to convert TF model to ML GeNN
+    if args.converter == 'few-spike':
```

Comment:
You could add a helper to args containing this logic to avoid copying it in all the examples. Maybe something like:
```python
def build_converter(self, norm_data, K=8, norm_time=500):
    if self.converter == 'few-spike':
        return FewSpike(K=K, norm_data=[norm_data])
    elif args.converter == 'data-norm':
        converter = DataNorm(norm_data=[norm_data], input_type=self.input_type)
    elif args.converter == 'spike-norm':
        return SpikeNorm(norm_data=[norm_data], norm_time=norm_time, input_type=self.input_type)
    else:
        return Simple(input_type=self.input_type)
```